be/src/main/java/com/jjikmuk/sikdorak/store/domain/Address.java

```diff
@@ -1,14 +1,18 @@
 package com.jjikmuk.sikdorak.store.domain;
 
+import com.jjikmuk.sikdorak.store.exception.InvalidAddressException;
+import java.util.Objects;
 import javax.persistence.Column;
 import javax.persistence.Embeddable;
 import lombok.AccessLevel;
+import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Getter
 @Embeddable
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
```

Comment:
```suggestion
@Builder(builderMethodName = "hiddenBuilder")
```

builder() 메서드가 외부에서 노출되니 이름으로 사용하지 말라고 명시해주는건 어때요?

Reply:
hiddenBuilder() 빌더를 아예 없애고 싶었지만, 

아래 예시에서 나온 `@Builder(builderMethodName = "hiddenBuilder", access = AccessLevel.PRIVATE)` 은 작동하지 않더라구요. 제 생각엔 이게 최선인거같아요

https://stackoverflow.com/questions/29885428/required-arguments-with-a-lombok-builder